common/elasticsearch/validator/queryValidator.go

```diff
@@ -77,17 +78,20 @@ func (qv *VisibilityQueryValidator) validateListOrCountRequestForQuery(whereClau
 		whereClause := strings.TrimSpace(whereClause)
 		// #nosec
 		if common.IsJustOrderByClause(whereClause) { // just order by
-			placeholderQuery = "SELECT * FROM dummy " + whereClause
+			placeholderQuery = fmt.Sprintf("SELECT * FROM dummy %s", whereClause)
```

Comment:
What's the difference compare to the old one?

Reply:
not much difference but might be more proper to some security tool.